modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc

```diff
@@ -0,0 +1,88 @@
+= Proxy Migration
+
+In {productname} 4.3, the containerized proxy is managed by a set of systemd services.
+
+In {productname} {productnumber} management of the containerized proxy was re-designed and made simpler with the [command]``mgrpxy`` tool.
+
+This section will help you migrate from the legacy [systemitem]``systemd`` proxy using the new [command]``mgrpxy`` tool.
+
+[IMPORTANT]
+====
+An in-place migration from {productname} 4.3 to 5.0 will remain unsupported due to the HostOS change from {sles} 15 SP4 to {micro} {microversion}
+====
+
+
+
+== Migrate from a Legacy Proxy to a Containerized Proxy with Systemd
+
+=== Generate the Proxy Configuration
+
+.Task: Generate the Proxy Configuration
+. Log in to the {productname} Server {webui}.
+. Select menu:Systems[Proxy Configuration] from the left navigation.
+. Enter your Proxy FQDN. Use the same FQDN as the original proxy host.
+. Enter your Server FQDN.
+. Enter the Proxy port number. __We recommend using the default port of 8022__
+. Certificate and private key are located on the Server container host in `/var/lib/containers/storage/volumes/root/_data/ssl-build/`.
+  * RHN-ORG-TRUSTED-SSL-CERT
+  * RHN-ORG-PRIVATE-SSL-KEY
+. Copy the certificate and key to your machine with: 
++
+
+----
+scp root@suma43-example.com:/root/ssl-build/RHN-ORG-PRIVATE-SSL-KEY .
+scp root@suma43-example.com:/root/ssl-build/RHN-ORG-TRUSTED-SSL-CERT .
+----
+
+. Select btn:[Choose File] and browse your local machine for the certificate.
+. Select btn:[Choose File] and brose your local machine for the private key.
+. Enter the CA password.
+. Click btn:[Generate].
+
+=== Transfer the Proxy Configuration to the New Proxy Host
+
+.Task: Transfer the Proxy Configuration
+// Add details on this transfer, filename and where it is located. `/root/ssl-build/`
```

Comment:
The layout for this is broken when being rendered:
![image](https://github.com/uyuni-project/uyuni-docs/assets/12104291/794f8686-cb48-435d-940a-f3e89a63f85d)